tests/core/utils/test_helpers.py

```diff
@@ -21,3 +22,6 @@ def test_optional(self):
 
         self.assertEqual(optional(user, default=0).non_existing_attr, 0)
         self.assertEqual(optional(user, default=0).non_existing_method(), 0)
+
+    def test_url(self):
+        self.assertEqual(url.url("upload"), "http://localhost:8000/upload")
```

Comment:
Here I need to do that ...
We would like to be able to do `url("upload")`...

It's because `url` is the instance bound to the container of UrlHelper and then we have to call the method...

Reply:
Or maybe what we should bind is the url() method of UrlHelper directly ... ?

Reply:
That works too I think lol

Reply:
It's working in the shell but does not seem to work in the unit test :/